src/go/k8s/api/redpanda/v1alpha2/redpanda_clusterspec_types.go

```diff
@@ -548,16 +564,6 @@ type TieredConfig struct {
 	CloudStorageUploadCtrlUpdateIntervalMs *int `json:"cloud_storage_upload_ctrl_update_interval_ms,omitempty"`
 }
 
-// TieredStoragePersistentVolume configures configurations for a PersistentVolumeClaim to use to store the Tiered Storage cache.
-type TieredStoragePersistentVolume struct {
-	// Adds annotations to the PersistentVolumeClaims to provide additional information or metadata that can be used by other tools or libraries.
-	Annotations map[string]string `json:"annotations,omitempty"`
-	// Applies labels to the PersistentVolumeClaims to facilitate identification and selection based on custom criteria.
-	Labels map[string]string `json:"labels,omitempty"`
-	// Specifies the StorageClass for the PersistentVolumeClaims to determine how PersistentVolumes are provisioned and managed.
-	StorageClass *string `json:"storageClass,omitempty"`
```

Comment:
So, it looks like this field is not in the `PersistentVolume` struct that you're leveraging instead of this struct.

Reply:
Nice catch! I wonder why this didn't get picked up by the test. TBF the problem space is massive. I'll try cutting out some tests for sub-structs and see where that gets to.

Reply:
Nope, it's there (link from this commit) https://github.com/redpanda-data/redpanda-operator/blob/9235d32835a7ead298a9f572faeb09fe53482a19/src/go/k8s/api/redpanda/v1alpha2/redpanda_clusterspec_types.go#L578

Reply:
oh gosh, I'm sorry, I was reading an unfortunately collapsed diff poorly 🤦

Reply:
No apologies needed, I'd rather double check something than accidentally break something.